### PR TITLE
[ADD] account_lock_constrains: add new module making the lock dates i…

### PR DESCRIPTION
**Pull request by smetl**
…rreversible

-task: https://www.odoo.com/web#id=38178&view_type=form&model=project.task&action=333&active_id=967&menu_id=4720

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
@smetl you also need to generate the .pot for the new module

File: addons/account_lock_constrains/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    'name' : 'Irreversible Lock Date',
+    'version' : '1.0',
+    'category': 'Accounting',
+    'description': """
+    Make the lock date irreversible:
+
+    - The lock date for advisors must be higher of equals of the lock date for users.
+    - The lock date for advisors must be higher of the last day of previous month.
+    - The lock date for advisors must be strictly higher of the previous one.
```
Comment (qdp-odoo) [inline addons/account_lock_constrains/__openerp__.py]: error messages are wrong and should be fixed as suggested by Laura:
You cannot define stricter conditions on advisors than on users. Please make sure that the lock date on advisor is not set after the lock date for users.
You cannot lock a period that is not finished yet. Please make sure that the lock date for advisors is not set after the last day of the previous month.
The new lock date for advisors must be set after the previous lock date.

Please adapt the docstring, the error messages and this module's description accordingly

File: addons/account_lock_constrains/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name' : 'Irreversible Lock Date',
```
Comment (antonylesuisse) [inline addons/account_lock_constrains/__openerp__.py]: REname module to account_lock

**Comment by tde-banana-odoo** [comment]
@qdp-odoo Actually I think we don't care about translations and pot files in master as we will generate them before v12 release and make a call for translation at that time. Is it still the working like that @mart-e  ?

**Comment by qdp-odoo** [comment]
new module account_lock landed in v9, at revision https://github.com/odoo/odoo/commit/2eb344f23b3a9daa8e7c7ddaead145a8b05b39bf

File: addons/account_lock/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
```
Comment (nhomar) [inline addons/account_lock/__openerp__.py]: the file should not be called **manifest**?

Comment (wtaferner) [reply addons/account_lock/__openerp__.py]: not in 9.0 afair :-)

Comment (nhomar) [reply addons/account_lock/__openerp__.py]: iups!

**Pull request closed without merging**